bin/phyx2owl.js

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const phyx = require('..');
+
+/*
+ * An application for converting input Phyx files to OWL ontologies.
+ */
+
+// Read command line arguments.
+const argv = require('yargs')
+  .usage("$0 [files or directories to convert into OWL ontologies]")
+  .describe('max-internal-specifiers', 'The maximum number of internal specifiers (phylorefs with more than this number will be ignored)')
+  .default('max-internal-specifiers', 8)
+  .describe('max-external-specifiers', 'The maximum number of external specifiers (phylorefs with more than this number will be ignored)')
+  .default('max-external-specifiers', 8)
+  .help()
+  .alias('h', 'help')
+  .argv
+
+/*
+ * Get a list of all files in a directory. We will recurse into directories and choose
+ * files that meet the criteria in the function `check(filename) => boolean`.
+ */
+function getFilesInDir(filePath, check = (filename => filename.toLowerCase().endsWith(".json"))) {
+  // console.debug(`Processing file: ${dir}`)
```

Comment:
There are a few more comments of this sort below. The `{$dir}` won't work anymore if uncommented, right? So either these commented out lines should be fixed, or removed.

Reply:
Done in b1d2562. I ended up fixing them rather than removing them, since they might be useful in debugging in the future.